routing/dht/dht.go

```diff
@@ -322,7 +322,7 @@ func (dht *IpfsDHT) betterPeersToQuery(pmes *pb.Message, p peer.ID, count int) [
 	// == to self? thats bad
 	for _, p := range closer {
 		if p == dht.self {
-			log.Debug("Attempted to return self! this shouldnt happen...")
+			log.Error("Attempted to return self! this shouldnt happen...")
```

Comment:
`log.Error` impacts the end user. we should no longer be using `log.Error`. use `log.Debug` or `log.Event`